README.md

```diff
@@ -34,6 +34,11 @@ Or run tests with:
 npm test
 ```
 
+You can also generate seed data script via:
+```sh
+NODE_ENV=test npx knex seed:run
+```
+
```

Comment:
not sure what seed data does or facilitates? is there anything to add here to clarify?